### DISCUSSION TITLE
Mantine Fileinput does not send file in request

### DISCUSSION THREAD

**Discussion by iam-sachinverma** [mantinedev/mantine#6300] [Q&A] [answered]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.9.0

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

All

### Describe the bug

```
const BaseForm = ({
  initialValues,
  onSubmit,
}) => {
  const form = useForm({
    initialValues: {
      ...initialValues,
      image: [],
    },
    validate: {},
  });

  return (
    <form
      className="adventure-form"
      encType="multipart/form-data"
      onSubmit={form.onSubmit(() => {
        onSubmit(form.values);
      })}
    >
    
      <SimpleGrid cols={2} spacing="lg" verticalSpacing="lg">

        <FileInput
          label="Cover Image"
          placeholder="Upload cover image file"
          {...form.getInputProps("image")}
        />

      </SimpleGrid>

      <div className="button-wrapper">
        <Button variant="gradient" fullWidth type="submit" mt="xl" size="md">
          Submit
        </Button>
      </div>
    </form>
  );
};

export default BaseForm;

```

Describe the bug
I'm using Vite with React 18, and for file uploads I was using FileInput. the submitted request FormData has a value object for image but . In the attached picture you can see submissions.
![Screenshot 2023-03-23 234950](https://github.com/mantinedev/mantine/assets/83693262/ca6bdf75-0fae-420f-aeec-2da73987b0d6)

The console.log does log the proper file with name and other values, but submitting the form doesn't have log value it only have empty obj {}

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment, marked as answer]
In your screenshot I see the File object which is what FileInput does. I do not see any issue here.
